### PR TITLE
Client: Build deployment

### PR DESCRIPTION
**Pull request by yuval-hazaz**

### CODE REVIEW

File: packages/amplication-client/src/VersionControl/Build.tsx [hunk 1]
```diff
@@ -238,6 +234,48 @@ const Build = ({ build, onError, open }: Props) => {
...
+          />
+
+          <span>{build.status}</span>
+        </>
+      )}
+      <span className="spacer" />
+      <UserAndTime account={account} time={build.createdAt} />
+    </div>
+  );
+};
```
Comment (iddan) [inline packages/amplication-client/src/VersionControl/Build.tsx:277]: Can you extract this to another file?

File: packages/amplication-server/src/core/build/build.service.spec.ts [hunk 2]
```diff
@@ -161,6 +162,9 @@ const EXAMPLE_ACTION_STEP = {
   id: 'EXAMPLE_ACTION_STEP_ID'
 };
 
+
+const deploymentFindManyMock = jest.fn();
```
Comment (iddan) [inline packages/amplication-server/src/core/build/build.service.spec.ts:166]: Please add a test that checks this mock.

**Review by iddan — COMMENTED** [review]
Good job, see comments.

File: packages/amplication-client/src/VersionControl/Build.tsx (hunk 1 above)
Comment (yuval-hazaz) [reply packages/amplication-client/src/VersionControl/Build.tsx:277]: This component will never be used without the build component and it is even not exported. Do we really want to have it in a separate file?

Comment (iddan) [reply packages/amplication-client/src/VersionControl/Build.tsx:277]: Yes, any component more than a few lines long better exist in a component inside the directory of the main component.

Comment (iddan) [reply packages/amplication-client/src/VersionControl/Build.tsx:277]: This is not a show stopper. Please handle whenever you have the time.

**Pull request merged**